.github/workflows/docker-build-push.yml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2024 Kaushlendra Pratap <kaushlendra-pratap.singh@siemens.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+name: Build and Push Docker Image
+
+on:
+  push:
```

Comment:
Should we add a release trigger as well? Furthermore, is the project ready to start versioning and releasing? cc: @GMishx

Reply:
Yea, I also thought about it. But wasnt sure with the release cycle :D

Reply:
Lets do that once we have a release cycle in place :-)